[2024-04-26T22:51:10.152+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-26T22:51:10.173+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Crypto_ETL.concat_datasets manual__2024-04-26T07:33:07.181560+00:00 [queued]>
[2024-04-26T22:51:10.183+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Crypto_ETL.concat_datasets manual__2024-04-26T07:33:07.181560+00:00 [queued]>
[2024-04-26T22:51:10.185+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-04-26T22:51:10.201+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): concat_datasets> on 2024-04-26 07:33:07.181560+00:00
[2024-04-26T22:51:10.209+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=605) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-26T22:51:10.210+0000] {standard_task_runner.py:63} INFO - Started process 607 to run task
[2024-04-26T22:51:10.211+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Crypto_ETL', 'concat_datasets', 'manual__2024-04-26T07:33:07.181560+00:00', '--job-id', '131', '--raw', '--subdir', 'DAGS_FOLDER/processing_ETL.py', '--cfg-path', '/tmp/tmp86u7czdv']
[2024-04-26T22:51:10.212+0000] {standard_task_runner.py:91} INFO - Job 131: Subtask concat_datasets
[2024-04-26T22:51:10.259+0000] {task_command.py:426} INFO - Running <TaskInstance: Crypto_ETL.concat_datasets manual__2024-04-26T07:33:07.181560+00:00 [running]> on host 7542c54372fc
[2024-04-26T22:51:10.357+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Crypto_ETL' AIRFLOW_CTX_TASK_ID='concat_datasets' AIRFLOW_CTX_EXECUTION_DATE='2024-04-26T07:33:07.181560+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-26T07:33:07.181560+00:00'
[2024-04-26T22:51:10.358+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-26T22:51:10.408+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-26T22:51:10.409+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-26T22:51:10.422+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Crypto_ETL, task_id=concat_datasets, execution_date=20240426T073307, start_date=20240426T225110, end_date=20240426T225110
[2024-04-26T22:51:10.469+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-26T22:51:10.499+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-26T22:51:10.502+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
